api/controllers/posts.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+
+const Post = require('../models/Post')
+
+// "show a particular post" route
+async function goToPost(req, res) {
+    try {
+        const post = await Post.findById(parseInt(req.params.id))
+        res.status(200).json(post)
+    } catch(err) {
+        res.status(404).json({err})
+    }
+}
+
+// "add a post" route
+async function newPost(req, res) {
+    try {
+        console.log(req.body);
+        const post = await Post.create(req.body.title, req.body.date, req.body.name, req.body.body)
```

Comment:
Textbook use of controllers.  Well done!  Just wondering if it might have been more readable/less confusing to call the body of your post something other than 'body'?